src/css/styles.scss

```diff
@@ -3,8 +3,9 @@
 
 :root {
   --zIndex-canvas: 1;
-  --zIndex-wysiwyg: 2;
-  --zIndex-layerUI: 3;
+  --zIndex-interactiveCanvas: 2;
```

Comment:
The rendering order needs to be: 

1. Render static scene (due to shape cache)
2. Render interactive canvas

Therefore we need to bring interactive canvas above the static one above with the z-index.